### ISSUE TITLE
[Home page] → Community → Code of conduct

### ISSUE THREAD

**Issue by kaktusus** [FreeCAD/FreeCAD-translations#293]
No translations for the menu label and the entire page.

![obraz](https://github.com/FreeCAD/FreeCAD-translations/assets/39754351/d787bfa0-a68e-4cbb-a5fc-5a679e9c3977)

**Comment by chennes** [comment]
This appears to also be true of the "Forum" and "Calendar" links, right? The pages themselves will not be translated as they simply import content from the actual MD files in the FreeCAD GitHub repo, but we can definitely translate the link text.

**Comment by kaktusus** [comment]
So in order to ensure the display of translated texts, we need to prepare Markdown files in the remaining languages.

**Comment by yorikvanhavre** [comment]
What we can also do is translate the link itself, so when translations are made available, it's just a matter to change the link to the translated version. But problem: Where to store the translated code of conducts? Inside the FreeCAD source code?

**Comment by kaktusus** [comment]
Can these files be in a remote location? E.g. the GitHub platform?

**Comment by yorikvanhavre** [comment]
Yes. At the moment the contributors guidelines are on github...
Maybe these could actually be copied to here, and all translations be placed in a folder. I'm just wary of the possibility to one be updated without the other...
Maybe we could place that code of conduct on crowdin? Then we could script something to retrieve the translations...

That would also be a good experiment to test how markdown files work on crowdin...

**Comment by kaktusus** [comment]
Good idea, it will be an introduction to the translation of documentation.
Here you can learn practically about the possibilities:
https://crowdin.com/project/freecad-asm3-wiki
https://github.com/Kuzma30/FreeCAD_assembly3/wiki

It is a pity that our colleague @Kuzma30 is not with us, he prepared this project.

Edit:
A quick insight.
[vokoscreenNG-2024-03-04_16-11-18.webm](https://github.com/FreeCAD/FreeCAD-translations/assets/39754351/95caee14-d59b-46d5-a572-d0c62246bd83)

**Comment by kaktusus** [comment]
hey :stuck_out_tongue_winking_eye:
but you already have :arrow_down: your test project ....
https://crowdin.com/project/freecad-documentation

**Comment by kaktusus** [comment]
Done !!
